src/chapters/orm-relationships/many-to-one.rst

```diff
@@ -0,0 +1,205 @@
+
+.. figure:: figures/find-usages.png
+   :alt: The search results pane after running Find Usages on EventType
+   :width: 600px
+
+   All remaining references to ``EventType`` in our project
+
+The first occurrence is in ``EventController.displayCreateEventForm``:
+
+.. sourcecode:: java
+   :lineno-start: 35
+
+   model.addAttribute("types", EventType.values());
+
+This line passes a collection of all of the values of ``EventType`` into the view, to be rendered in the form used to create new events.
+
+Since we are now using ``EventCategory`` to group events, our code should instead be passing in all of the category objects in our app. To fetch category objects, we need an instance of ``EventCategoryRepository`` in our controller. Add an autowired instance to the top of the controller:
+
+.. sourcecode:: java
+   :lineno-start: 24
+
+   @Autowired
+   private EventCategoryRepository eventCategoryRepository;
+
+Now, use the repository to fetch all saved categories:
+
+.. sourcecode:: java
+   :lineno-start: 40
+
+   model.addAttribute("categories", eventCategoryRepository.findAll());
+
+This line replaces the line references ``EventType.values()``. Notice that we have relabeled this attribute ``"categories"`` to be more consistent. This also requires updating the ``events/create.html`` template:
+
+.. sourcecode:: html
+   :lineno-start: 27
+
+   <div class="form-group">
+      <label>Category
+      <select th:field="${event.eventCategory}">
+         <option th:each="eventCategory : ${categories}"
+                  th:value="${eventCategory.id}"
+                  th:text="${eventCategory.name}"
+         ></option>
+      </select>
+      <p class="error" th:errors="${event.eventCategory}"></p>
+      </label>
+   </div>
+
+This new template code includes several changes:
+
+#. The ``select`` is now bound to the field ``eventCategory``.
+#. The loop references ``categories`` and has an updated iterator variable name, ``eventCategory``.
+#. The value of the ``select`` is now ``eventCategory.id``. This allows model binding to occur upon form submission. Spring Boot will determine the category object to assign to the new event object by referencing the ``id``.
+#. The display text for each ``option`` is now ``eventCategory.name``
+#. The error message display now references the new field, ``event.eventCategory``.
+
+The remaining usages of ``EventType`` refer to:
+
+#. An ``Event`` constructor parameter.
+#. The getter/setter pair for ``type`` in ``Event``.
+#. An import statement in ``EventController``.
+#. The ``EventType`` class itself.
+
+Removing this unneeded code resolves all remaining compiler errors. 
+
+Finally, the ``events/create.html`` template needs to be updated as well, since it still contains a reference to the ``type`` field of ``Event``:
```

Comment:
The "Finally" and "as well" are redundant here.  You can get by with one or the other.